pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -227,6 +224,11 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return fmt.Errorf("log-storage-controller failed to watch primary resource: %v", err)
 	}
 
+	err = c.Watch(&source.Kind{Type: &operatorv1.Authentication{}}, &handler.EnqueueRequestForObject{})
```

Comment:
We're always including Authentication in Calico Enterprise installs right?

Reply:
Yes

Reply:
yes